tidb-cloud/migrate-from-mysql-using-data-migration.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Migrate MySQL-Compatible Databases to TiDB Cloud Using Data Migration
+summary: Learn how to migrate data from MySQL-compatible databases hosted in Amazon Aurora MySQL, Amazon Relational Database Service (RDS), or a local MySQL instance to TiDB Cloud using Data Migration.
+---
+
+# Migrate MySQL-Compatible Databases to TiDB Cloud Using Data Migration
+
+This document describes how to migrate data from a MySQL-compatible database on a cloud provider (Amazon Aurora MySQL and Amazon Relational Database Service (RDS)) or on-premises to TiDB Cloud directly using the Data Migration feature of the TiDB Cloud console.
+
+This feature allows you to migrate your application data and its ongoing changes to TiDB Cloud (either in the same region or cross regions) within a short downtime window.
+
+## Limitations
+
```

Comment:
Please help indicate that this Data Migration feature is not available for Serverless Tier clusters.

Reply:
ok added.